HACKING.md

```diff
@@ -289,11 +304,13 @@ specify its name in an argument:
 
 ### Adding a new report
 
- 1. Create new module in */lib/MusicBrainz/Server/Report/*.
- 2. Add created module into [ReportFactory.pm](https://github.com/metabrainz/musicbrainz-server/blob/master/lib/MusicBrainz/Server/ReportFactory.pm).
+ 1. Create new module in `/lib/MusicBrainz/Server/Report/`.
```

Comment:
Additionally, the leading `/` can be removed since all paths are relative to the `musicbrainz-server` directory. It was confusing with the `/lib` directory of the FHS.

```suggestion
 1. Create new module in `lib/MusicBrainz/Server/Report/`.
```